src/WebJobs.Script/azurefunctions/functions.js

```diff
@@ -70,16 +70,20 @@ function createFunction(f) {
                 callback(err);
             }
             else {
-                var values = {};
                 if (context.res && context.bindings.res === undefined) {
                     context.bindings.res = context.res;
                 }
+
+                var bindingValues = {};
                 for (var name in context.bindings) {
-                    values[name] = context.bindings[name];
+                    bindingValues[name] = context.bindings[name];
                 }
-                context.bind(values, function (err) {
```

Comment:
Previously we were calling back into .NET here to allow it to process these bindings.